SimPEG/EM/NSEM/source.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
+import scipy.sparse as sp
+
+from ... import maps
+from ...utils import mkvc
+from ..FDEM.source import BaseFDEMSrc
+from ..Utils import omega
+from .Utils.sourceUtils import homo1DModelSource
+
+#################
+###   Sources ###
+#################
+
+# class BaseFDEMSrc(FDEMBaseSrc):
```

Comment:
@grosenkj: It doesn't look to me that this class is adding or changing any functionality from the BaseFDEM source, so I plan to remove it, and instead directly inherit from FDEM. Any concerns with that?